pkg/utils/k8sutil.go

```diff
@@ -210,6 +210,9 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 	labels := map[string]string{
 		"function": name,
 	}
+	for k, v := range funcObj.Metadata.Labels {
```

Comment:
This ordering allows `--label function=muhaha` to override the usual `function=name` label - should that be expected/allowed?